go.mod~

```diff
@@ -0,0 +1,103 @@
+module github.com/openshift/sdn
```

Comment:
This is the backup file created by... emacs? :)

Reply:
bad `git add .` !